tablemusthave/table.py

```diff
@@ -48,9 +51,16 @@ def from_csv(cls, f, null_values=default_null_values, **csv_reader_args):
         rows = csv.reader(f, **csv_reader_args)
         colnames = next(rows)
         return cls(colnames, rows, null_values)
+    
+    @classmethod
+    def from_data(cls, data, null_values=default_null_values):
+        return cls(data.keys(), list(zip(*[data[col] for col in data.keys()])), null_values)
```

Comment:
You might be packing too much action into one line here...

Reply:
Just added a docstring to explain it, can break up the line if you still think it's too much.